app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, counter_cache: :comments_counter
+
```

Comment:
- [OPTIONAL] Kindly make use of one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_comments_count method every time a new comment has been saved to database, before you can make the update_comment_count method private